src/CoffeeMatch.js

```diff
@@ -0,0 +1,23 @@
+//how to get image from www.impactroasters.dk/coffee/ and add it to the array of matches
```

Comment:
I can see that it's jpg files on the website, so maybe let's have the png files in this repo after all. I'll comment on the other file how we should do that.